src/component/label.js

```diff
@@ -15,13 +15,15 @@ const balanceStyles = StyleSheet.create({
   numeral: {
     fontFamily: 'WorkSans ExtraLight',
     fontSize: font.sizeXXXL,
-    lineHeight: font.lineHeightXXXL,
+    lineHeight: null,
+    marginLeft: 10,
+    marginRight: 10,
   },
   unit: {
     fontFamily: 'WorkSans Regular',
     fontSize: font.sizeL,
-    lineHeight: 60,
-    marginLeft: 10,
```

Comment:
This `marginLeft: 10` is still required if `marginRight` on the numeral is removed.

Reply:
fixed :)